test/users.test.js

```diff
@@ -0,0 +1,93 @@
+const request = require('supertest');
+const truncateDatabase = require('./setup');
+const app = require('../app');
+
+beforeAll(async () => {
+  process.env.NODE_ENV = 'test';
+  await truncateDatabase();
+});
+
+describe('POST users', () => {
+  const newUserData = {
+    name: 'William',
+    lastName: 'Salazar',
+    email: 'william.salazar@wolox.co',
+    password: 'thisIsAVerySecurePassword'
+  };
+
+  test('create new user', async done => {
```

Comment:
you must agrupate singup test cases using [describe()](https://jestjs.io/docs/en/api#describename-fn). And name   test cases more specifically. for example "Should signup a new valid user".

Reply:
Done